src/pages/Editor.vue

```diff
@@ -62,8 +63,8 @@
       </div>
 
       <!--- input grid -->
-      <div class="grid grid-rows-6 grid-cols-1 m-5 justify-start">
-        <div class="row-start-1 row-span-1 grid gap-y-4">
+      <div class="flex flex-col m-5 justify-start">
+        <div class="grid gap-y-4">
```

Comment:
fixed the breaking css of dropdown when you removed the test values div.
Using a grid here was inherently a problem. Things stretched out unnecessarily if we added > 10 options. Flex works better here and everything snaps into place